Link title attribute for items with icons

zzap opened the pull request:
When icon is added to item its link has icon's class as title attribute, while without icon it's item's title. Link's title should remain item's title, regardless of icon being included or not.

At the moment it is like this:

```
<a href="" title="fa-star-o">
	<i class="fa fa-star-o" aria-hidden="true"></i>&nbsp;Title
</a>
```

And it should be like this:

```
<a href="" title="Title">
	<i class="fa fa-star-o" aria-hidden="true"></i>&nbsp;Title
</a>
```

zzap committed:
Link title attribute for items with icons

When icon is added to item its link has icon's class as title attribute, while without icon it's item's title. Link's title should remain item's title, regardless of icon being included or not.

At the moment it is like this:

```
<a href="" title="fa-star-o">
	<i class="fa fa-star-o" aria-hidden="true"></i>&nbsp;Title
</a>
```

And it should be like this:

```
<a href="" title="Title">
	<i class="fa fa-star-o" aria-hidden="true"></i>&nbsp;Title
</a>
```

zzap committed:
Travis fixes.

pattonwebz commented:
Hey Milana,

Just this weekend I decided to retire the Bootstrap 3 version of the walker in favour of an updated version that's built for Bootstrap 4. The new version treats icons how they should be treated :)

That said it is still useful to pull this change into the V3 branch as it's a useful fix for anyone still using BS3 and you already wrote the code to PR :)

I'll look at adding this with a version bump and changelog later today when I'm at my computer :)

zzap commented:
Yeah, I moved to Bootstrap 4 too (and modified Walker to fit some specific cases). And then I got request for old theme, still using v.3, so I thought to do a quick PR while I still have file open :D.

> Just this weekend I decided to retire the Bootstrap 3 version of the walker in favour of an updated version that's built for Bootstrap 4. The new version treats icons how they should be treated :)

Keep up the good work. I might be adding some PRs to v4 as my client is very creative and heavily relies on Bootstrap :+1: 

Cheers

pattonwebz commented:
Hey, so with me planning to move along with a v4 version I just pushed a version for v3 at the weekend that I was considering final... and then you made a PR with a bugfix so I wasn't quite sure what to do haha!

I don't want the master branch to diverge before we move the v4 walker into it so I have made a new branch - [v3-branch](https://github.com/wp-bootstrap/wp-bootstrap-navwalker/tree/v3-branch) - where I'm going to keep the stable version of the walker for Bootstrap 3. 

I've already merged your PR into that branch and bumped the version number so I'm going to close this PR but it's technically already merged.

Thank you for fixing this bug and sharing the fix. I look forward to any PRs that you are able to make for either version in future :)
